airbyte-webapp/src/components/JobItem/components/MainInfo.tsx

```diff
@@ -116,30 +118,43 @@ const MainInfo: React.FC<MainInfoProps> = ({
   };
 
   const jobStatus = getJobStatus(job);
+  const streamsToReset = (job as JobsWithJobs).job.resetConfig?.streamsToReset;
   const isNotCompleted =
     jobStatus === JobStatus.pending || jobStatus === JobStatus.running || jobStatus === JobStatus.incomplete;
 
-  const jobStatusLabel = isPartialSuccess ? (
-    <FormattedMessage id="sources.partialSuccess" />
-  ) : (
-    <FormattedMessage id={`sources.${getJobStatus(job)}`} />
-  );
+  const getLabel = () => {
+    if (streamsToReset) {
+      const formattedMessageId = streamsToReset.length % 10 === 1 ? "sources.streamReset" : "sources.streamsReset";
```

Comment:
Pluralization can't be put in code like this (due to several reasons, the main one being different languages supporting different amount of plurals). The pluralization should happen completely within the actual message in `en.json`. Have a look at other `plural` examples in there and this documentation: https://support.crowdin.com/icu-message-syntax/#plural